config/config_helpers.go

```diff
@@ -157,6 +165,33 @@ func pathRelativeToInclude(include *IncludeConfig, terragruntOptions *options.Te
 	return util.GetPathRelativeTo(currentPath, includePath)
 }
 
+// Return the relative path from the current Terragrunt configuration to the included Terragrunt configuration file
+func pathRelativeFromInclude(include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
+	if include == nil {
+		return ".", nil
+	}
+
+	relativePath, err := findInParentFolders(terragruntOptions)
+	if err != nil {
+		return "", errors.WithStackTrace(err)
+	}
+
+	return filepath.ToSlash(filepath.Dir(relativePath)), nil
+}
+
+// Return the AWS account id associated to the current set of credentials
+func getAWSAccountID(terragruntOptions *options.TerragruntOptions) (string, error) {
+	iamSession := iam.New(session.Must(session.NewSession()))
+
+	result, err := iamSession.ListUsers(&iam.ListUsersInput{MaxItems: aws.Int64(1)})
```

Comment:
Nice idea, we never realized that this call existed and alway relied on iam. Thank you!